notebooks/text_models/solutions/keras_for_text_classification.ipynb

```diff
@@ -29,23 +29,25 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.                Input(shape=(1,), dtype=tf.string),
</pre><p> </p><p><code>dtype='string'</code> in Keras. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/654/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>